BonnAnalysis/Dual.lean

```diff
@@ -40,27 +40,26 @@ structure IsConjExponent (p q : ℝ≥0∞) : Prop where
 namespace IsConjExponent
 
 lemma symm (hpq : p.IsConjExponent q) : q.IsConjExponent p := by
-    rw [isConjExponent_iff, add_comm, hpq.inv_add_inv_conj]
+  simp [isConjExponent_iff, add_comm, hpq.inv_add_inv_conj]
```

Comment:
how is this an improvement? `rw`/`simp_rw` are preferred over `simp`.

Reply:
Ops, sorry. You're totally right.

Reply:
I'll change it as soon as I get access to my PC.

Reply:
Done